### ISSUE TITLE
`libopus error: corrupted stream` error with `.ogg` / `.opus` when playing with `mpd`, but not when playing with `mpv`/`vlc`

### ISSUE THREAD

**Issue by fireofearth** [MusicPlayerDaemon/MPD#569]
## Question

I'm getting this error when I try to play some `.opus` music files in `mpd` that I downloaded with `youtube-dl`. This also happens with `.ogg` music files.
```
May 29 17:50 : exception: nested: libopus error: corrupted stream
```
However I can play the music files in `mpv` and fresh installed `vlc` just fine so I'm guessing it may have something to do with `mpd`. The above error doesn't give much detail so I'm not sure if it's a problem with `mpd`, or if it's with a plugin it uses then I'm not sure which one. Below are the ones that I have. May you please advise? Thanks!

```
Decoders plugins:
 [mad] mp3 mp2
 [mpg123] mp3
 [vorbis] ogg oga
 [oggflac] ogg oga
 [flac] flac
 [opus] opus ogg oga
 [sndfile] wav aiff aif au snd paf iff svx sf voc w64 pvf xi htk caf sd2
 [audiofile] wav au aiff aif
 [dsdiff] dff
 [dsf] dsf
 [hybrid_dsd] m4a
 [faad] aac
 [mpcdec] mpc
 [wavpack] wv
 [modplug] 669 amf ams dbm dfm dsm far it med mdl mod mtm mt2 okt s3m stm ult umx xm
 [mikmod] amf dsm far gdm imf it med mod mtm s3m stm stx ult uni xm
 [wildmidi] mid
 [fluidsynth] mid
 [ffmpeg] 16sv 3g2 3gp 4xm 8svx aa3 aac ac3 adx afc aif aifc aiff al alaw amr anim apc ape asf atrac au aud avi avm2 avs bap bfi c93 cak cin cmv cpk daud dct divx dts dv dvd dxa eac3 film flac flc fli fll flx flv g726 gsm gxf iss m1v m2v m2t m2ts m4a m4b m4v mad mj2 mjpeg mjpg mka mkv mlp mm mmf
mov mp+ mp1 mp2 mp3 mp4 mpc mpeg mpg mpga mpp mpu mve mvi mxf nc nsv nut nuv oga ogm ogv ogx oma ogg omg opus psp pva qcp qt r3d ra ram rl2 rm rmvb roq rpl rvc shn smk snd sol son spx str swf tak tgi tgq tgv thp ts tsp tta xa xvid uv uv2 vb vid vob voc vp6 vmd wav webm wma wmv wsaud wsvga wv wve
 [gme] ay gbs gym hes kss nsf nsfe sap spc vgm vgz
 [pcm]

Filters:
 libsamplerate soxr

Tag plugins:
 id3tag

Output plugins:
 shout null fifo pipe alsa ao oss openal solaris pulse jack httpd recorder

Encoder plugins:
 null vorbis opus lame twolame wave flac

Archive plugins:
 [bz2] bz2
 [zzip] zip
 [iso] iso

Input plugins:
 file archive alsa tidal qobuz curl ffmpeg smbclient nfs mms cdio_paranoia
```

**Comment by MaxKellermann** [comment]
How to reproduce?

**Comment by edlman** [comment]
I encountered the same problem. I solved this by disabling 'opus' plugin in mpd.conf
```
decoder {
    plugin "opus"
    enabled "no"
}
```
As the opus format is basicaly ogg format I tried to rename the `audio.opus` file to `audio.ogg` and mpd was able to play it via different plugin (I'm not sure which one possibly `vorbis` or `oggflac`). After disabling `opus` plugin mpd plays opus audio via other plugin probably `ffmpeg` as I don't see other plugin able to play opus audio. (see output of `mpd --version`).
So the problem with playing opus files lies in the opus plugin. To test and debug it try it on opus audio downloaded from youtube using linux cli
`# youtube-dl -x https://www.youtube.com/watch?v=SIFQu629Udg` (The Interrupters - Take Back The Power)
